pkg/kn/commands/flags/eventtype.go

```diff
@@ -25,6 +25,6 @@ type EventtypeFlags struct {
 func (e *EventtypeFlags) Add(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&e.Type, "type", "t", "", "Cloud Event type")
 	cmd.Flags().StringVar(&e.Source, "source", "", "Cloud Event source")
-	cmd.Flags().StringVarP(&e.Broker, "broker", "b", "", "Cloud Event broker")
+	cmd.Flags().StringVarP(&e.Broker, "broker", "b", "", "Cloud Event Broker. This flag is added for the convenience, since Eventing v1beta2 brokers as represented as KReference type.")
```

Comment:
I think this clarification is more confusing than it helps. I don't think that the user know ad hoc what a "KReference type" is. For the help message it would be better to use something like "Deprecated. Please use --reference instead", or if its not deprecated, I just would keep it as is. Anything else is just confusing,.